contribs/gnofaucet/README.md

```diff
@@ -22,14 +22,25 @@ Start the faucet. This repository provides middleware for integrating GitHub OAu
 
     ./build/gnofaucet serve github -chain-id dev -mnemonic "source bonus chronic canvas draft south burst lottery vacant surface solve popular case indicate oppose farm nothing bullet exhibit title speed wink action roast" --github-client-id=<CLIENT_ID> --cooldown-period=24h --max-claimable-limit=100000000 (100 gnot)
 
```

Comment:
should we explicitly say to start a redis ?

Reply:
not related to this PR. Could you please open a separate issue for this? Thanks.